pkgs/by-name/ga/galene/package.nix

```diff
@@ -37,9 +41,35 @@ buildGoModule (finalAttrs: {
 
   passthru = {
     tests.vm = nixosTests.galene.basic;
-    updateScript = nix-update-script {
-      extraArgs = [ "--version-regex=galene-(.*)" ];
+    updateScriptSrc = gitUpdater {
+      rev-prefix = "galene-";
     };
+    updateScriptVendor = writeShellApplication {
+      name = "update-galene-vendorHash";
+      runtimeInputs = [
+        nix
+        sd
+      ];
+      text = ''
+        export UPDATE_NIX_ATTR_PATH="''${UPDATE_NIX_ATTR_PATH:-galene}"
+
+        oldhash="$(nix-instantiate . --eval --strict -A "$UPDATE_NIX_ATTR_PATH.goModules.drvAttrs.outputHash" | cut -d'"' -f2)"
+        newhash="$(nix-build -A "$UPDATE_NIX_ATTR_PATH.goModules" --no-out-link 2>&1 | tail -n3 | grep 'got:' | cut -d: -f2- | xargs echo || true)"
+
+        if [ "$newhash" == "" ]; then
+          echo "No new vendorHash."
+          exit 0
+        fi
+
+        fname="$(nix-instantiate --eval -E "with import ./. {}; (builtins.unsafeGetAttrPos \"version\" $UPDATE_NIX_ATTR_PATH).file" | cut -d'"' -f2)"
+
+        ${sd.meta.mainProgram} --string-mode "$oldhash" "$newhash" "$fname"
```

Comment:
I think we use `lib.getExe` instead nowadays?

Reply:
Just dropped the "smart" meta lookup and hardcoded `sd`, more consistent with the rest of the script(s).